tests/integration/test_sidecar_upgrade.py

```diff
@@ -167,13 +177,21 @@ async def build_and_upgrade_fixture(
     charm = await ops_test.build_charm(".")
     # Most of the times the integration tests will fail due to timeout without the force flag.
     app: juju.application.Application = ops_test.model.applications[application_name]
+    wordpress_revision_before = await get_app_revision(application_name)
+    upgrade_from_ingress = wordpress_revision_before <= 14
```

Comment:
This is a "magic number", can you define it in a constant instead?

Reply:
Updated, thanks!